js_modules/dagster-ui/packages/ui-core/src/owners/css/DefinitionOwners.module.css

```diff
@@ -0,0 +1,4 @@
+.user_display {
```

Comment:
Comment above re: `BaseTag`. We should also stick to `camelCase` for classnames.